pkg/provider/aws/aws.go

```diff
@@ -87,6 +88,20 @@ func New(log *logger.FunLogger, env v1alpha1.Environment, cacheFile string) (*Pr
 	if envRegion := os.Getenv("AWS_REGION"); envRegion != "" {
 		region = envRegion
 	}
+	sha := os.Getenv("GITHUB_SHA")
+	// short sha
+	if len(sha) > 8 {
+		sha = sha[:8]
+	}
+	actor := os.Getenv("GITHUB_ACTOR")
+	branch := os.Getenv("GITHUB_REF_NAME")
+	repo_name := os.Getenv("GITHUB_REPOSITORY")
```

Comment:
It is better to include the full repo name that includes the GitHub org. In the future, it is possible that we have other GitHun orgs than `NVIDIA/`
```suggestion
	repoName := os.Getenv("GITHUB_REPOSITORY")
```